recipe/meta.yaml

```diff
@@ -47,14 +43,12 @@ outputs:
         - python
         - cycler >=0.10
         - fonttools >=4.22.0
-        - freetype >=2.3
         - kiwisolver >=1.0.1
         - {{ pin_compatible('numpy') }}
         - packaging >=20.0
         - pillow >=6.2.0
         - pyparsing >=2.2.1
         - python-dateutil >=2.7
-        - tk  # [linux]
```

Comment:
Same here, why was tk removed?
If I understand correctly, because ** backend** is not set in build_base.sh conda_mpl_config.cfg

Reply:
Again, not sure about this: the `run_exports` from `tk` are explicitly ignored (this was always in the recipe), because presumably the headers are already included in `matplotlib` and the runtime symbols are provided by Python itself. So I don't understand why we'd do that and then include `tk` as a runtime dependency.
Anyway the last time I tried building this with `tk`, I got an `overdepending check failed` error; I'll try again.

Reply:
yeah you are right, it doesn't look to be needed.
https://github.com/matplotlib/matplotlib/blob/main/setupext.py#L442